conda_smithy/cli.py

```diff
@@ -272,6 +273,13 @@ class Rerender(Regenerate):
     else:
         args = parser.parse_args()
 
+    # Check conda version for compatibility
+    conda_version_tuple = tuple(map(int, conda.__version__.split('.')[:3]))
+    if conda_version_tuple > (4, 2):
```

Comment:
`distutils` has some version comparison stuff that might be helpful. Would check that out.